include/alp/structures.hpp

```diff
@@ -592,6 +593,36 @@ namespace alp {
 			using inferred_structures = tuple_cat< std::tuple< LowerTrapezoidal >, Trapezoidal::inferred_structures >::type;
 		};
 
+		template<>
+		struct isInstantiable< LowerTrapezoidal, LowerTrapezoidal > {
+			template< typename ImfR, typename ImfC >
+			static bool check( const ImfR &imf_r, const ImfC &imf_c ) {
+				(void) imf_r;
```

Comment:
A basic condition would probably be `imf_c.map( 0 ) <= imf_r.map( imf_r.n - 1 )` ?